workers/constructors/SharedWorker/SharedWorker-constructor.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<title>Test SharedWorker constructor functionality.</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+test(() => {
+  assert_throws_js(Error,
+                   function() {
+                     new SharedWorker({toString:function(){throw new Error()}})},
+                   'toString exception should be propagated');
+}, 'Test toString propagation exception.');
+
+test(() => {
+  try {
+    var foo = {toString:function(){new SharedWorker(foo)}};
+    new SharedWorker(foo);
+  } catch(ex) {
+    assert_true(true, 'trying to create Sharedworkers recursively should result in an: ' +
+                ex + 'exception.')
+  }
+}, 'Test recursive SharedWorker creation.');
```

Comment:
I don't think this is testing something that's in the standard. JS doesn't define stack overflow exceptions, does it?

Reply:
You're correct, this test was copied from the Worker tests which originally came from chromium's test suite 

https://github.com/web-platform-tests/wpt/commit/810ec024edb749071534252c66c11ddd8afd5c48#diff-37b01470c4b3da94d56a74c80cab5ae2e79c78b4aa3b876f23546205c406cc00R14-R22

I will remove the test from this pull-request :+1: 
Should we also remove it from the Worker tests @annevk ?

Reply:
I've removed this test from SharedWorker 👍